2025-02-24 12:15:20,097 ERROR: Exception on /send_message/earg [GET] [in C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\app\routes.py", line 229, in send_message
    return render_template('send_message.html', title=('Send Message'),
                           form=form, recipient=recipient)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 771, in compile
    self.handle_exception(source=source_hint)
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\student2\Desktop\p\app\templates\send_message.html", line 5, in template
    <h1>{{ ('Send Message to %(recipient)%', recipient=recipient) }}</h1>
    ^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.TemplateSyntaxError: expected token ')', got '='
2025-02-24 12:15:29,649 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:15:54,709 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:16:11,654 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:16:13,134 ERROR: Exception on /send_message/earg [GET] [in C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\app\routes.py", line 229, in send_message
    return render_template('send_message.html', title=('Send Message'),
                           form=form, recipient=recipient)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 771, in compile
    self.handle_exception(source=source_hint)
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\student2\Desktop\p\app\templates\send_message.html", line 5, in template
    <h1>{{ 'Send Message to 'recipient }}</h1>
    ^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got 'recipient'
2025-02-24 12:16:24,587 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:16:33,345 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:17:01,438 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:17:03,330 ERROR: Exception on /send_message/earg [GET] [in C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py:1414]
Traceback (most recent call last):
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 2190, in wsgi_app
    response = self.full_dispatch_request()
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1486, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1484, in full_dispatch_request
    rv = self.dispatch_request()
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\app.py", line 1469, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask_login\utils.py", line 290, in decorated_view
    return current_app.ensure_sync(func)(*args, **kwargs)
           ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\app\routes.py", line 229, in send_message
    return render_template('send_message.html', title=('Send Message'),
                           form=form, recipient=recipient)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    template = app.jinja_env.get_or_select_template(template_name_or_list)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 1087, in get_or_select_template
    return self.get_template(template_name_or_list, parent, globals)
           ~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 1016, in get_template
    return self._load_template(name, globals)
           ~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 975, in _load_template
    template = self.loader.load(self, name, self.make_globals(globals))
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\loaders.py", line 138, in load
    code = environment.compile(source, name, filename)
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 771, in compile
    self.handle_exception(source=source_hint)
    ~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\student2\Desktop\p\venv\Lib\site-packages\jinja2\environment.py", line 942, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\student2\Desktop\p\app\templates\send_message.html", line 5, in template
    <h1>{{ 'Send Message to %(us)', us=recipient }}</h1>
    ^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.TemplateSyntaxError: expected token 'end of print statement', got '='
2025-02-24 12:17:28,732 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:18:41,099 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
2025-02-24 12:19:03,371 INFO: Microblog startup [in C:\Users\student2\Desktop\p\app\__init__.py:49]
